mfaust35/translation_dummy_mf
# Feature add user model

+ Add model User with a UserDao and test
+ Upgrade model translation: add foreign key userId, upgrade dao and
tests
+ TEMPORARILY have a DEFAULT_USER_ID used in the application until a
login feature exists
+ Input activity now create translation linked to user1
+ Select activity display the name on current user
+ Add DatabaseVersion2 file containing details of migration toward
version 2 all well as details of database creation in version 2. Other
version upgrade should have a similar file DatabaseVersionX. Also test
migrations.
+ Add a baseDaoTest to setup and tearDown the database for all
xxxDaoTest
+ Change room dependency in build.gradle to use the one in jetpack
+ Add assertJ dependency
